xarray/core/common.py

```diff
@@ -1263,11 +1266,24 @@ def where(self, cond, other=dtypes.NA, drop: bool = False):
 
         return ops.where_method(self, cond, other)
 
+    def set_close(self, close_store: Optional[Callable[[], None]]) -> None:
```

Comment:
Maybe just call this argument `close` rather than `close_store`? In principle, this is very generic and not "store" specific.

Reply:
Agreed. Done.